api/swagger.yml

```diff
@@ -3431,11 +3431,88 @@ components:
+              * `12` - ErrGettingReceiverAccount error
+              * `13` - ErrCantSendToEthAddr error
+              * `14` - ErrNotAtomicTxsInPostPoolTx error
+              * `15` - ErrFailedToGetCurrentBlock error
+              * `16` - ErrFailedToGetAuctionVars error
+              * `17` - ErrFailedToAddEmptySlot error
+              * `18` - ErrTxsNotAtomic error
+              * `19` - ErrSingleTxInAtomicEndpoint error
+              * `20` - ErrRqOffsetOutOfBounds error
+              * `21` - ErrInvalidAtomicGroupID error
+              * `22` - ErrFailedToFindOffsetToRelativePositionCode error
+        type:
+          type: string
+          enum:
+            - ErrParamValidationFailed
+            - ErrDuplicatedKey
+            - ErrSQLTimeout
+            - ErrSQLNoRows
+            - ErrExitAmount0
+            - ErrInvalidTxTypeOrTxID
+            - ErrFeeOverflow
+            - ErrGettingSenderAccount
+            - ErrAccountTokenNotEqualTxToken
+            - ErrInvalidNonce
+            - ErrInvalidSignature
+            - ErrGettingReceiverAccount
+            - ErrCantSendToEthAddr
+            - ErrNotAtomicTxsInPostPoolTx
+            - ErrFailedToGetCurrentBlock
+            - ErrFailedToGetAuctionVars
+            - ErrFailedToAddEmptySlot
+            - ErrTxsNotAtomic
+            - ErrSingleTxInAtomicEndpoint
+            - ErrInvalidAtomicGroupID
+            - ErrFailedToFindOffsetToRelativePosition
+          description: >
+            Error types:
+              * `ErrParamValidationFailed` - this error appears when user sending wrong info in query|uri|form params
+              * `ErrDuplicatedKey` - record already exists in db
+              * `ErrSQLTimeout` - node is under heavy pressure and sql connection timeout happened
+              * `ErrSQLNoRows` - there is no such item in database
+              * `ErrExitAmount0` - receiving (and rejecting) a tx of type exit with amount 0 in /transactions-pool request
+              * `ErrInvalidTxTypeOrTxID` - tx with invalid type or txID was sent in /transactions-pool request
+              * `ErrFeeOverflow` - tx with fee overflow was sent in /transactions-pool request
+              * `ErrGettingSenderAccount` - sender account for tx was not found in /transactions-pool request
+              * `ErrAccountTokenNotEqualTxToken` - account token is not equal tx token in /transactions-pool request
+              * `ErrInvalidNonce` - invalid nonce was sent in /transactions-pool request
+              * `ErrInvalidSignature` - invalid signature was sent in /transactions-pool or /account-creation-authorization requests
+              * `ErrGettingReceiverAccount` - receiver account for tx was not fount in /transactions-pool request
+              * `ErrCantSendToEthAddr` - it's not possible to send to eth addr when there is an idx not associated to the Eth addr and token or the coordinator hasn't an authorization to create a valid account
```

Comment:
it's not possible to send to Eth address when there is no account index associated to this Eth address and token, or the coordinator hasn't got an authorization to create an account